modules/@apostrophecms/schema/index.js

```diff
@@ -361,6 +361,11 @@ module.exports = {
       }
     });
 
+    self.addFieldType({
+      name: 'radio',
+      extend: 'select'
```

Comment:
I don't see any `vueComponent` property, is this complete for the front end in some way I'm not seeing?

Reply:
Never mind I forgot there's a default naming pattern.